OutOfSchool/OutOfSchool.RazorTemplatesData/OutOfSchool.RazorTemplatesData.csproj

```diff
@@ -8,4 +8,8 @@
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OutOfSchool.DataAccess\OutOfSchool.DataAccess.csproj" />
+  </ItemGroup>
```

Comment:
(Possible design issue, but I might misinterpret it, so please correct me if I'm wrong)
`RazorTemplatesData` project for me seems to be part of presentation level. 
`DataAccess` project - seems to represent persistence level.
If we have classical layered architecture - this two should never have direct communication.

Please uncover why you need this link in first place, and please reconsider, shouldn't related logic be moved to `WebApi` project?

Reply:
@VadymLevkovskyi, I removed the reference to the DataAccess project, separated email templates, and moved the related logic to the WebAPI. Thank you!